.eslintrc.cjs

```diff
@@ -55,6 +55,21 @@ module.exports = {
     // so that floating-promises can be explicitly permitted with void operator
     'no-void': ['error', { allowAsStatement: true }],
 
+    // The rule is “safe await separator" which implements the architectural
+    // goal of “clearly separate an async function's synchronous prelude from
+    // the part that runs in a future turn”. That is our architectural rule. It
+    // can be trivially satisfied by inserting a non-nested `await null` in an
+    // appropriate place to ensure the rest of the async function runs in a
+    // future turn.  “sometimes synchronous" is a bug farm for particularly
+    // pernicious bugs in which you can combine two correct pieces of code to
+    // have emergent incorrect behavior.  It’s absolutely critical for shared
+    // service code. That means contracts, but it also means kernel components
+    // that are used by multiple clients. So we enable it throughout the repo
+    // and aim for no exceptions.
+    //
+    // The default is 'warn', but we want to enforce 'error'.
+    '@jessie.js/safe-await-separator': 'error',
```

Comment:
The `@jessie.js/recommended` rules are the ones that understand `// @jessie-check` in all JS files, and also add warnings for `@jessie.js/safe-await-separator` in all JS files.  When you're actually writing Jessie code, the much stricter `@jessie.js/no-nested-await` is enabled as an `error` rule.

I couldn't bring myself to create an entirely new eslint-plugin just for this rule, and I justified the coupling with Jessie by making it a recommended warning (gently prompting users of the eslint plugin to upgrade it to `error` or downgrade it to `off`).

Reply:
I do think it is worth making both of these changes to Jessie. 
   * IMO the `@jessie.js/no-nested-await` should default to 'off', but would like your thoughts.
   * ***IF*** `@jessie.js/no-nested-await` is downgraded from `error`, then `@jessie.js/safe-await-separator` should be upgraded to `error`. ELSE no change suggested.

But since Jessie is a separate repo, this PR does not need to wait on that Jessie change.